### DISCUSSION TITLE
Scrolling in PDF viewer using mouse wheel doesn't work anymore

### DISCUSSION THREAD

**Discussion by klotzoliver** [cefsharp/CefSharp#3658] [General]
If a PDF is opened in CefSharp (WPF) the mouse wheel does work as expected.
But after clicking on scrollbar for page preview the mouse wheel doesn't work anymore in PDF content area.

**Comment by amaitland** [comment]
Please download and test with the CEF Sample application (cefclient).

https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

**Reply by klotzoliver** [reply]
App downloaded and tested. Same behavior as described at the top.

**Reply by amaitland** [reply]
You'll need to report the problem on https://bitbucket.org/chromiumembedded/cef/issues?status=new&status=open

Please include a detailed step by step list of instructions to reproduce the problem.

**Reply by amaitland** [reply]
Reported at https://bitbucket.org/chromiumembedded/cef/issues/3153/mouse-wheel-in-chromium-pdf-viewer-doesnt

Anyone else experiencing the problem should follow on bitbucket.
